### DISCUSSION TITLE
Aqara Smart Radiator Thermostat E1 Battery consumption

### DISCUSSION THREAD

**Discussion by domdal** [Koenkk/zigbee2mqtt#15004] [Devices]
I have the Aqara Smart Radiator Thermostat E1 in combination with Home Assistant and Better Thermostat.
Since i have the thermostat, i have already changed the batteries 2 times. 
The last one lasted about 2 weeks.
How long do your batteries last?

I'm trying to figure out where the high energy consumption is coming from, because the thermostat has only been turned on twice.
Does anyone have an idea?

**Comment by auanasgheps** [comment]
Hi, I know you posted a long time ago, but I have your exact setup.

My E1 drains batteries in about two months, which I think is still a short period. Have you got any update on your issue?

**Comment by glagla1** [comment]
Have exact same pb. I found on the aqara e1 docs that its is not advised to use rechargeable batteries. Maybe the deain come from this !!

**Comment by fhloston** [comment]
I have three of these. In one which was running with first battery for more than a year the replacement batteries now drain within days. The other two do not show this behaviour.

Replacement batteries were always new alkaline batteries. I have now run through 5 sets very quickly.

![image](https://github.com/user-attachments/assets/98d3d6d4-f2f8-4d9e-8296-32462981aedd)

**Reply by fhloston** [reply]
I tried to forget this device in my main zigbee setup, replaced batteries, paired it to my debug coordinator and then re-paired it to the main setup.

After doing that, device shows no firmware version in z2m and upon update check it shows update available - even though it was previously already on latest 0.0.0_1030.

![image](https://github.com/user-attachments/assets/511de086-9863-4790-9bf6-2bbc51e667f3)
![image](https://github.com/user-attachments/assets/08957cba-5d85-412c-b9c2-8403de813801)

After the update it still shows no version and after update check it shows update available again.

I could go in a loop now :sweat: 

After a while and a bit of clicking around in exposes and hitting interview it changes again:
![image](https://github.com/user-attachments/assets/1005444a-fcae-4537-8872-4da635bd73e5)
![image](https://github.com/user-attachments/assets/c3ab887f-b8b4-4f1b-9237-d66382581b7c)

Now update throws errors:

```[2025-01-16 09:52:31] error: 	z2m: Publish 'set' 'child_lock' to 'Bedroom Thermostat' failed: 'Error: Expected boolean, got: string'
[2025-01-16 10:05:55] error: 	z2m: Publish 'set' 'child_lock' to 'Bedroom Thermostat' failed: 'Error: Expected boolean, got: string'
[2025-01-16 10:20:02] error: 	z2m: Update of 'Bedroom Thermostat' failed ([0x54ef44100067341f | lumi.airrtc.agl001] Failed to find an endpoint which supports the OTA cluster)
[2025-01-16 10:23:46] error: 	z2m: Update of 'Bedroom Thermostat' failed ([0x54ef44100067341f | lumi.airrtc.agl001] Failed to find an endpoint which supports the OTA cluster)
[2025-01-16 10:23:46] error: 	z2m: Update of 'Bedroom Thermostat' failed ([0x54ef44100067341f | lumi.airrtc.agl001] Failed to find an endpoint which supports the OTA cluster)```

**Reply by auanasgheps** [reply]
The firmware I have is 0.0.0_1030, dated  Feb 20 2023, like the one you have installed.
The issue you're facing is only with the 0030 and not 1030, right?

To be honest, this winter the issue did not occur to me.
![immagine](https://github.com/user-attachments/assets/0e04e7ef-7f58-4127-8998-3da1876635ac)

New good batteries installed on 9th November, now at 53%. I think it's doing good!

A lot has changed in BT from last year, so we'll never know what really caused the issue. (at least for me)

These are my settings for BT:

Calibration Mode: AI Time Based
Additional Settings: Ignore all inputs on the TRV like a child lock

My temperature sensor reports temperature changes every 0.2°C, so I didn't set any tollerance. If you have a sensor that is very precise, you might want to increase the tollerance, to reduce unnecessary movements. Try setting "target_temp_step" to 0.2°C or above.
